DataFormats/PatCandidates/interface/Jet.h

```diff
@@ -512,7 +514,19 @@ namespace pat {
       /// Subjet collection names
       std::vector<std::string> const & subjetCollectionNames() const { return subjetLabels_; }
 
-
+      /// Access to mass of subjets
+      double groomedMass(unsigned int index = 0) const{
+	return nSubjetCollections() > 0 ?
+	  std::accumulate( subjets(0).begin(), subjets(0).end(),
+			   reco::Candidate::LorentzVector(), [] (reco::Candidate::LorentzVector a, reco::CandidatePtr const & b){return a + b->p4();}).mass() :
+	  -1.0;
+      }
+      double groomedMass(std::string label) const{
```

Comment:
please use `const std::string &` instead of `std::string`
(it should be changed also in the other methods, BTW)